includes/definitions/discovery/socomec.yaml

```diff
@@ -0,0 +1,72 @@
+mib: SOCOMECPDU-MIB
+modules:
+    os:
+        features: SOCOMECPDU-MIB::dpduIdentModel.0
```

Comment:
Seems this Oid should be the hardware instead?

Reply:
I was not sure what to specify here. used as an example panduit.yaml - there was a branch with a model.

Reply:
If you look at the data it says;
"hardware": "00",
"features": "NRT-OP-PDU1-28"

So hardware seems to just be a revision, not so interesting, and the feature is the model

Reply:
tried to fix it.